decentraland/explorer-desktop-launcher
# fix: appx for windows store

- Added the correct appx store logo images and created the needed folder structure for Electron building process to include those in the built AppX
- Fixed issue preventing AppX launcher to execute the desktop client from the correct path

### How to test AppX (Windows Store App) launcher
1. Uninstall the Desktop Client
2. Enable Windows PC for development: https://docs.microsoft.com/en-us/windows/apps/get-started/enable-your-device-for-development ( Open windows menu, search: Developer Settings and press Developer Mode
3. Install Windows SDK: https://developer.microsoft.com/en-us/windows/downloads/windows-sdk/
4. Add the Windows SDK executables path to your Windows Environment PATH variable, in my case the patth to add was `C:\Program Files (x86)\Windows Kits\10\bin\10.0.22000.0\x64\`. To be able to use the `MakeAppx` command from the CMD.
5. Download AppX from this branch: https://renderer-artifacts.decentraland.org/launcher-branch/chore_change-publisher-name/Decentraland.appx
6. Open CMD as administrator and use the following command to unpack the Appx (needed to be installed later, since it's unsigned): `MakeAppx unpack /p "C:/Decentraland.Appx" /d "C:/DCLAppx/"`. Make sure to replace the base path to the place you downloaded the `Decentraland.appx` file
7. Open Powershell as administrator and run the following command to install the unpacked AppX from that location: `Add-AppxPackage -Path C:/DCLAppx/AppxManifest.xml -Register`. After that you should see the Decentraland application installed and with the Decentraland icon.
8. Open the Decentraland application and confirm that the client opens after continuing as guest or logging in.

### How to test our regular distributed launcher
1. Uninstall the Desktop Client
9. Download the launcher from this branch: https://renderer-artifacts.decentraland.org/launcher-branch/chore_change-publisher-name/Install%20Decentraland.exe
10. Open it and log-in or enter as guest
